docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -70,6 +70,10 @@ Start a Strapi application with autoReload disabled.
 This command is there to run a Strapi application without restarts and file writes (aimed at production usage).
 Certain features are disabled in the `strapi start` mode because they require application restarts.
 
+- **strapi start --no-migrate**<br/>
+  Skips migrations which are normally run when the application boots up to sync content-types and any user-generated migrations. If `--no-migrate` option is passed, migrations must be run separately with the `strapi migrate` command.
```

Comment:
```suggestion
  Skip migrations which are normally run when the application boots up to sync content-types and any user-generated migrations. If `--no-migrate` option is passed, migrations must be run separately with the `strapi migrate` command.
```

I noticed we have inconsistencies in this documentation 😅  I'll make sure that every verb is in imperative form.